README.md

```diff
@@ -18,15 +24,16 @@ The OpenTok Node SDK provides methods for:
 
 If you are looking for the JavaScript Client SDK please see the [@opentok/client](https://www.npmjs.com/package/@opentok/client) NPM module.
 
-## Note!
+## Before you start
 
-This library is designed to work with the Tokbox/OpenTok platform, part of the Vonage Video API. If you are looking to use the Vonage Video API and using the Vonage Customer Dashboard, you will want to install the [Vonage Server SDK for Node](https://github.com/vonage/vonage-node-sdk), which includes support for the Vonage Video API.
+This library is designed to work with the TokBox/OpenTok platform, which is part of the Vonage Video API. **The TokBox platform is no longer accepting new sign-ups.** If you are a new customer or starting a new project, you should use the [Vonage Customer Dashboard](https://dashboard.vonage.com) and the [Vonage Server SDK for Node](https://github.com/vonage/vonage-node-sdk), which includes support for the Vonage Video API.
 
-Not sure which exact platform you are using? Take a look at [this guide](https://api.support.vonage.com/hc/en-us/articles/10817774782492).
+If you're an existing TokBox customer, don't worry — the TokBox platform continues to run on the same infrastructure as the Unified Vonage platform and remains supported. 
```

Comment:
This line is fine to add, but let's keep the other line as well. We get a lot of people accidentally pushed to this library from Google.

Reply:
```suggestion
If you are using the Tokbox platform, do not worry! The Tokbox platform is not going away, and this library will continue to be updated. While we encourage customers to check out the new Unified platform, there is no rush to switch. Both platforms run the exact same infrastructure, with the main difference being a unified billing interface and easier access to [Vonage’s other CPaaS APIs](https://www.vonage.com/communications-apis/). Do note that over time there may be some feature differences that will only exist on the Unified platform.
```